utils/docker/Dockerfile.centos.7

```diff
@@ -33,7 +33,7 @@ RUN set -e;                                                      \
     if [ -n "$REPO_FILE_URL" ]; then                             \
         dnf --quiet config-manager --disable epel;               \
     fi;                                                          \
-    dnf clean all
+    dnf clean all && dnf upgrade
```

Comment:
Each step should delete all temporary files to reduce the image sizes so this should go before the clean all.

Reply:
Good call.  Fixing.